Classes/Manager/Submanagers/OCTSubmanagerAvatars.m

```diff
@@ -38,11 +43,12 @@ - (void)setAvatar:(UIImage *)avatar
         [fileManager createDirectoryAtPath:[path stringByDeletingLastPathComponent]
                withIntermediateDirectories:YES
                                 attributes:nil
-                                     error:nil];
-        [data writeToFile:path atomically:NO];
+                                     error:error];
```

Comment:
If we cannot create directory we don't want to write file and send it to tox. It would be better to return here.